ui/apps/platform/src/Containers/Clusters/ClustersSidePanel.tsx

```diff
@@ -145,51 +152,28 @@ function ClustersSidePanel({ selectedClusterId, setSelectedClusterId }) {
                             switch (managerType(cluster)) {
                                 case 'MANAGER_TYPE_HELM_CHART':
                                     setMessageState({
-                                        type: 'warn',
-                                        message: (
-                                            <>
-                                                <h3 className="font-700 mb-2">
-                                                    Helm-managed cluster
-                                                </h3>
-                                                <p>
-                                                    This is an Helm-managed cluster. The settings of
-                                                    Helm-managed clusters cannot be changed here,
-                                                    please ask your DevOps team to change the
-                                                    settings by updating the Helm values.
-                                                </p>
-                                            </>
-                                        ),
+                                        variant: 'warning',
+                                        title: 'Helm-managed cluster',
+                                        text: 'This is an Helm-managed cluster. The settings of Helm-managed clusters cannot be changed here, please ask your DevOps team to change the settings by updating the Helm values.',
```

Comment:
Grammar nit outside of this PR:
```suggestion
                                        text: 'This is a Helm-managed cluster. The settings of Helm-managed clusters cannot be changed here, please ask your DevOps team to change the settings by updating the Helm values.',
```